public/app/features/explore/Table.tsx

```diff
@@ -44,7 +49,25 @@ export default class Table extends PureComponent<TableProps> {
       accessor: text,
       className: VALUE_REGEX.test(text) ? 'text-right' : '',
       show: text !== 'Time',
-      Cell: row => <span className={filterable ? 'link' : ''}>{row.value}</span>,
+      Cell: row => (
+        <div>
+          {row.value}
+          {filterable && (
+            <span className="filter-links">
+              <Tooltip content="Filter for value" placement="bottom">
+                <a className="link" data-operator="=">
```

Comment:
The Table should be a generic viewer of data and should not prescribe what operator to use for the filter. I'd rather pass a constant string here like `INCLUDE` and let the datasource interpret that when it needs to modify the query.